README.md

```diff
@@ -23,3 +23,25 @@ You can see examples of these actions in use in the following repos:
 - [signal-desktop](https://github.com/snapcrafters/signal-desktop/main/.github/workflows)
 - [mattermost-desktop](https://github.com/snapcrafters/mattermost-desktop/main/.github/workflows)
 - [discord](https://github.com/snapcrafters/discord/main/.github/workflows)
+
+## Contributing
+
+If you'd like to contribute to this repository, please feel free to fork and create a pull request.
+
+There are a few style guidelines to keep in mind:
+
+- Code should be linted using [Prettier](https://prettier.io/). You can achieve this with `make
+lint` and `make format`. The only requirements are [`npx`](https://www.npmjs.com/package/npx) and
```

Comment:
FYI: Not critical, but I really dislike manual line wrapping on markdown files.  It can also create issues depending on where the markdown is used, with what formatter, and what markdown you're specifically modifying. Manual wrapping and markdown tables doesn't play well, for example. Same with code blocks.

Modern IDEs are smart enough to soft-wrap markdown files. In this case, it's difficult for me to quickly scan through and see whether you've missed a bullet or whether something is just manually wrapped text. PR diffs become very noisy because a one-word change can suddenly impact an entire block of text.